diff --git a/android/build.gradle b/android/build.gradle
index 3b8f2970c5045bba292e38b6c36708ab05b5f4f5..f7e42ee34d43561435031ea86df5e481663ed35d 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -41,6 +41,6 @@ dependencies {
     implementation "com.facebook.react:react-native:+"
 }
 task copyDownloadableDepsToLibs(type: Copy) {
-    from configurations.compile
+    from configurations.implementation
     into 'libs'
 }
