diff --git a/node_modules/react-native-odoo-promise-based/lib/index.js b/node_modules/react-native-odoo-promise-based/lib/index.js
index 6de9b33..66b37f0 100644
--- a/node_modules/react-native-odoo-promise-based/lib/index.js
+++ b/node_modules/react-native-odoo-promise-based/lib/index.js
@@ -25,6 +25,7 @@ Odoo.prototype.connect = function () {
 
   var options = {
     method: 'POST',
+    credentials: 'include',
     headers: {
       'Content-Type': 'application/json',
       'Accept': 'application/json',
@@ -183,10 +184,10 @@ Odoo.prototype._request = function (path, params) {
   var url = `${this.protocol}://${this.host}:${this.port}${path || '/'}`
   var options = {
     method: 'POST',
+    credentials: 'include',
     headers: {
       'Content-Type': 'application/json',
-      'Accept': 'application/json',
-      'Cookie': this.cookie
+      'Accept': 'application/json'
     },
     body: JSON.stringify({
       jsonrpc: '2.0',
