diff --git a/node_modules/react-native-sqlite-storage/src/android/build.gradle b/node_modules/react-native-sqlite-storage/src/android/build.gradle
index 6ad7d72..d3a1481 100644
--- a/node_modules/react-native-sqlite-storage/src/android/build.gradle
+++ b/node_modules/react-native-sqlite-storage/src/android/build.gradle
@@ -35,5 +35,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.14.+'
+    implementation 'com.facebook.react:react-native:0.14.+'
 }
