diff --git a/node_modules/typeorm/browser/util/StringUtils.js b/node_modules/typeorm/browser/util/StringUtils.js
index 24cabfc..dc19bfb 100644
--- a/node_modules/typeorm/browser/util/StringUtils.js
+++ b/node_modules/typeorm/browser/util/StringUtils.js
@@ -1,4 +1,4 @@
-import * as crypto from "crypto";
+import * as shajs from "sha.js";
 /**
  * Converts string into camelCase.
  *
@@ -86,7 +86,7 @@ export function shorten(input, options) {
  */
 export function hash(input, options) {
     if (options === void 0) { options = {}; }
-    var hashFunction = crypto.createHash("sha256");
+    var hashFunction = shajs.createHash("sha256");
     hashFunction.update(input, "utf8");
     var hashedInput = hashFunction.digest("hex");
     if (options.length) {
diff --git a/node_modules/typeorm/util/StringUtils.js b/node_modules/typeorm/util/StringUtils.js
index 537bc52..4b04d35 100644
--- a/node_modules/typeorm/util/StringUtils.js
+++ b/node_modules/typeorm/util/StringUtils.js
@@ -1,6 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-var crypto = require("crypto");
+var shajs = require("sha.js");
 /**
  * Converts string into camelCase.
  *
@@ -93,7 +93,7 @@ exports.shorten = shorten;
  */
 function hash(input, options) {
     if (options === void 0) { options = {}; }
-    var hashFunction = crypto.createHash("sha256");
+    var hashFunction = shajs.createHash("sha256");
     hashFunction.update(input, "utf8");
     var hashedInput = hashFunction.digest("hex");
     if (options.length) {
